cluster/examples/workloads/kubernetes/wordpress/gcp/provider.yaml

```diff
@@ -4,49 +4,17 @@ data:
   credentials.json: BASE64ENCODED_GCP_PROVIDER_CREDS
 kind: Secret
 metadata:
-  namespace: crossplane-system
+  namespace: gcp
   name: gcp-provider-creds
 type: Opaque
 ---
 apiVersion: gcp.crossplane.io/v1alpha2
 kind: Provider
 metadata:
-  namespace: crossplane-system
+  namespace: gcp
```

Comment:
I believe that we are wanting to move this to being `gcp-infra-dev`

Reply:
Let's keep this note and roll it into the next changeset